src/nvim/move.c

```diff
@@ -1345,20 +1309,15 @@ void scroll_cursor_top(int min_scroll, int always)
   new_topline = top + 1;
 
   /* count filler lines of the cursor window as context */
-  i = diff_check_fill(curwin, curwin->w_cursor.lnum);
-  used += i;
-  extra += i;
+  int extra = diff_check_fill(curwin, curwin->w_cursor.lnum);
+  used += extra;
```

Comment:
So.

i is here only for a temporary var that is equal to extra. Why? Look, extra is initialized during declaration to 0.
extra += i => extra = extra + i => extra = 0 + i => extra = i
If you look some lines below, you can see that i is set at the beginning of while. So the value of i here is not used anymore. 

I find it more readable (imho)
"extra" is "diff_check_fill" and we add it to "used"
than "i" is "diff_check_fill" and add it to "extra" and "used"